lib/utils/preference-manager.js

```diff
@@ -3,7 +3,13 @@ const prefDirectory = `${require('os').homedir()}/.autolabjs`;
 
 const cliConfig = new Preferences('autolabjs.cli.config', {}, {
   encrypt: false,
-  file: `${prefDirectory}/cli-config.json`,
+  file: `${prefDirectory}/prefs.json`,
```

Comment:
One observation. The preference-manager.js module has too many elementary getter / setter functions. Is it better to organize the work around manipulating preferences, credentials, locations json structures? Such a set of json objects and the methods would be more easy to reason about in the domain of AutolabJS.

Reply:
Also the code coverage for preference-manager.js needs to grow towards complete coverage.

Reply:
There is another way to handle preferences - 
There will be one getter and one setter
```Javascript
setPreference({
  name: 'preference_name',
  value: {
    option1: 'option1_value',
    option2: 'option2_value'
  }
});
```
And similarly for getter. Is this approach fine?

Reply:
Yup. Sounds good and is compact too.

The typical way to handle this is to retrieve the existing preferences, add/modify the needed values and save them again.